### ISSUE TITLE
bug report. log file getting too big.

### ISSUE THREAD

**Issue by xiangwei22** [PandoraPay/go-pandora-pay#10]
https://github.com/PandoraPay/go-pandora-pay/blob/debddf7c3b1c8b282b126976c90d2a2420322c33/gui/gui_logger/logger.go#L23-L30

the log file is only getting bigger. after many days it can crash the node. new files should be assigned every 24 hours.

**Comment by ibudisteanu** [comment]
You are right. The log file will only grow in case the node will not restart.

**Comment by xiangwei22** [comment]
any solution ?
